frontend/src/Admin/components/NavBar.js

```diff
@@ -35,6 +38,9 @@ export default class NavBar extends Component {
           <NavbarToggler onClick={this.toggle} />
           <Collapse isOpen={this.state.isOpen} navbar>
             <Nav className="ml-auto" navbar>
+              <NavItem>
+                <NavLink tag={Link} to='/' onClick={this.props.switchToJudge}>Switch to: Judge</NavLink>
```

Comment:
Quick thing,I think that the Switch to: Judge looks weird compared to the rest of the links on the navbar. No big deal if others disagree

Reply:
I think I would rename it to "View as Judge"